vaadin-checkbox-flow-parent/vaadin-checkbox-flow/src/test/java/com/vaadin/flow/component/checkbox/tests/CheckboxGroupTest.java

```diff
@@ -646,6 +647,12 @@ public void setAriaLabelledBy() {
         Assert.assertTrue(group.getAriaLabelledBy().isEmpty());
     }
 
+    @Test
+    public void implementsInputField() {
+        CheckboxGroup<String> field = new CheckboxGroup<String>();
+        Assert.assertTrue(field instanceof InputField);
```

Comment:
Maybe there is value in checking the type params for `InputField` as well?

Reply:
Probably I've added to all the tests